src/main/java/com/akademia/educators_back/to/CommentTo.java

```diff
@@ -0,0 +1,14 @@
+package com.akademia.educators_back.to;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CommentTo {
+
+    private Long id;
+    private String answer;
+    private UserTo userTo;
```

Comment:
Nie potrzebnie przechowujemy całego TO w innym TO. Tutaj wystarczy że Comment będzie miał id UserTo i od ProblemTO. Po id będziemy sobie wyciągać pozostałe